.github/workflows/cra.yaml

```diff
@@ -112,13 +114,21 @@ jobs:
 
   print-images:
     runs-on: ubuntu-latest
-    needs: [build-image]
+    needs: [build-image, build-test-image]
+    if: always() && contains(needs.*.result, 'success')
```

Comment:
Could you explain in a comment what is the intention of this line? 
Most of the team is not very familiar with gh actions and if were to change something in the area it could be difficult

Reply:
I can actually comment everything, so each setting is justified.
Would it make sense to add comments when I am applying fixes to rest of the pipelines, since if this change doesn't work it will be reverted?

Reply:
If there is any setting that could be potentially difficult to understand - then it's IMO worth adding a comment. For the sake PR I would like to request the explanation of just this one line :).